File: admin/get cli/e_bandobas/lib/app/jsondata/PointData/Point.dart
class Point {
  Point(
      {this.id,
      this.taluka,
      this.district,
      this.pointName,
      this.accessories,
      this.remarks, this.zoneName});

  Point.fromJson(dynamic json) {
    id = json['id'] ?? "0";
    taluka = json['taluka'] ?? "";
    district = json['district'] ?? "";
    pointName = json['pointName'] ?? "";
    accessories = json['accessories'] ?? "";
    remarks = json['remarks'] ?? "";
    zone = json['zone'] ?? 0;
    zoneName = json['zone-name'] ?? "";
  }
  num? id;
  String? taluka;
  String? district;
  String? pointName;
  String? accessories;
  String? remarks;
  num? zone;
  String? zoneName;

  Map<String, dynamic> toJson() {
    final map = <String, dynamic>{};
    map['id'] = id;
    map['name'] = taluka;
    map['district'] = district;
    map['pointName'] = pointName;
    map['accessories'] = accessories;
    map['remarks'] = remarks;
    map['zone'] = zone;
    map['zone-name'] = zoneName;
    return map;
  }
}

File: gujarat_police_backend/src/main/java/com/shyam/gujarat_police/entities/Point.java
package com.shyam.gujarat_police.entities;

import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import javax.validation.constraints.NotEmpty;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class Point extends BaseModel {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id", nullable = false)
    private Long id;

    @NotEmpty(message = "{validation.name.NotEmpty}")
    private String taluka;

    @NotEmpty(message = "{validation.name.NotEmpty}")
    private String district;

    @NotEmpty(message = "{validation.name.NotEmpty}")
    private String pointName;

//    @NotEmpty(message = "{validation.name.NotEmpty}")
    private String accessories;

//    @NotEmpty(message = "{validation.name.NotEmpty}")
    private String remarks;

    @ManyToOne
    @JoinColumn(name = "zone_id")
    private Zone zone;

    @JsonIgnore
    @OneToMany(mappedBy = "point", cascade = CascadeType.PERSIST)
    private List<AssignPolice> assignPolice;
}
